src/components/ui/Modal.tsx

```diff
@@ -28,7 +30,13 @@ export const Modal = ({
   };
 
   return (
-    <dialog ref={ref} className={styles.modalContainer} onClose={onClose}>
+    <dialog
+      ref={ref}
+      className={[styles.modalContainer, isSmall && styles.modalSmallContainer]
+        .filter((e) => !!e)
```

Comment:
could consider adding a package like this to avoid classnames like this, but not necessary - https://github.com/lukeed/clsx

<img width="195" alt="image" src="https://user-images.githubusercontent.com/16028371/214759812-a90cd6f9-50e9-408d-a5e7-cf01fc54b247.png">

Reply:
Well I don't know if adding a full new package just for that is a good idea

Reply:
Fine either way by me